### ISSUE TITLE
Data format of the input dataset

### ISSUE THREAD

**Issue by Hoclor** [AAshqar/DSCLRCN-PyTorch#5]
Thank you for fixing the google drive link. I am still trying to run the implementation, and am facing another issue now during the training (unmatched sizes of the labels and labels_sum when trying to calculate the loss). I suspect this is because the data is not input correctly.

Could you give some detail about how you converted the dataset from a set of images to the pickle files, or alternatively provide the code used to do this? As this is not provided, I have had to try to deduce how to do it by reverse-engineering your data_utils.py file, likely with errors.

Thank you for your help!

**Comment by Hoclor** [comment]
I have now resolved the above issue, but came across another: the readme file requests the images be resized to 92x128, but the model outputs results as 96x128 - I have assumed that the model is correct and the readme is wrong, could you confirm/correct this?

**Comment by tinaa160** [comment]
@Hoclor could you please share the pickle files or the code how you did that? I would be appreciated!
